### ISSUE TITLE
Upgrade to igdb API v4

### ISSUE THREAD

**Issue by QuentinSc** [mitchhymel/dart-igdb_client#6]
Hello,

Thank you for your work.
Any plans to migrate to api V4 ? 
https://api-docs.igdb.com/#upgrading-to-v4-from-v3
End of v3 this month :/

Thank you !

**Comment by mitchhymel** [comment]
Thanks for the heads up. I updated the package to use v4 api (8530680441e33f96b2a1117d60dc70ef17f326c0) and published the new version to pub [0.0.6](https://pub.dev/packages/igdb_client).

Since IGDB changed auth in v4 vs v3, I had to make some breaking changes to how to get an IGDB client - it's async now. [See the example](https://github.com/mitchhymel/dart-igdb_client/blob/master/example/igdb_client_example.dart#L8). But other than that, things should work the same as before.

Edit: Actually now that I reread the v4 documentation on usage in mobile apps, I realize I may rewrite how this library handles auth and publish a new version.

Second Edit: Updated again to 0.0.7, things should be working and the usage pattern is based on igdb documentation recommendation. Cheers!
